app/views/main_content_panel.py

```diff
@@ -2169,20 +2169,27 @@ def _check_git_repos_for_update(self, repo_paths: list[str]) -> None:
 
                         # Get the local and remote refs
                         local_ref = repo.head.reference
-                        refs = repo.refs()
-                        remote_ref = refs[f"origin/{local_ref.name}"]
+                        remote_ref = None
+                        for ref in repo.refs:
+                            if f"origin/{local_ref.name}" in ref.path:
+                                remote_ref = ref
+                                break
```

Comment:
(no changes needed) equivalent one-liner for (debatably) more readable code:
```
remote_ref = next((ref for ref in repo.refs if f"origin/{local_ref.name}" in ref.path), None)
```

Reply:
Thanks Should be good now